packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -824,10 +824,9 @@ export function mountResponderInstance(
   responderProps: Object,
   responderState: Object,
   instance: Instance,
-  rootContainerInstance: Container,
 ): ReactDOMEventResponderInstance {
   // Listen to events
-  const doc = rootContainerInstance.ownerDocument;
```

Comment:
I believe that we originally built the old event system around the  `rootContainerInstance` so that we had the option to move to a model where we attach the listeners on the `rootContainerInstance` instead of the `ownerDocument`. The idea was that if you attach it on the root, then it doesn't interfere with siblings trees and stopping propagation can stop it to non-React parents. That's why `rootContainerInstance` is passed around everywhere even though it doesn't really matter much which one is used (other than for portals into iframes).

Reply:
I know, but we never actually use it and it saves having to store the `rootContainerInstance` somewhere to be be passed in during the commit phase (which is why I removed it, as I couldn't see an easy way to get it in the commit phase again).